upbound_uxp/assets/dashboards/upbound_uxp_overview.json

```diff
@@ -0,0 +1 @@
th":4,"height":2}},{"id":8383428569535810,"definition":{"title":"Workqueue Adds Total","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"queries":[{"name":"query1","data_source":"metrics","query":"sum:uxp.workqueue_adds_total{*} by {host,pod,name}.as_count()"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":4,"y":8,"width":4,"height":2}},{"id":823949909374051,"definition":{"title":"Workqueue Work Duration Seconds Count","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"queries":[{"name":"query1","data_source":"metrics","query":"avg:uxp.workqueue_work_duration_seconds_bucket.count{*} by {host,pod,name,le}"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":8,"y":8,"width":4,"height":2}},{"id":5126533313330222,"definition":{"title":"Workqueue Unfinished Work Seconds","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"time":{},"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"queries":[{"name":"query1","data_source":"metrics","query":"avg:uxp.workqueue_unfinished_work_seconds{*}"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":0,"y":10,"width":4,"height":2}},{"id":4395817810242775,"definition":{"title":"Workqueue Depth","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"per_minute(query1)"}],"queries":[{"name":"query1","data_source":"metrics","query":"max:uxp.workqueue_depth{*}"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":4,"y":10,"width":4,"height":2}},{"id":6712236079650739,"definition":{"title":"Workqueue Work Duration Seconds Sum","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"queries":[{"name":"query1","data_source":"metrics","query":"sum:uxp.workqueue_work_duration_seconds_sum{*} by {host,pod,name}.as_count()"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":8,"y":10,"width":4,"height":2}}],"template_variables":[],"layout_type":"ordered","notify_list":[],"reflow_type":"fixed"}
```

Comment:
Can you also add a logo to this dashboard? It may also be helpful to add links to documentation in the description